camera:
  is_rectified: false
  # enter coordinates of camera origin with respect to quad origin
  translation: [0,0,0]
  # enter rotation matrix for quad to camera frame row-wise
  rotation: [0,-1,0,-1,0,0,0,0,-1]

box:
  minSize: 2000
  maxAreaIndex: 0.2
  maxEigenIndex: 1.069
  maxDiagIndex: 75
  centreCorrectIndex: 4
  maxCentreDist: 120

flags:
  areaCheck: true
  diagCheck: true
  eigenCheck: true
  sizeCheck: true
  centreCorrect: false
  debug: true
  verbose: false

# yellow
yellow:
  h_min: 20
  h_max: 40
  s_min: 80
  s_max: 255
  v_min: 80
  v_max: 255

#red
red:
  h_min: 0
  h_max: 20
  s_min: 80
  s_max: 255
  v_min: 80
  v_max: 255

#blue
blue:
  h_min: 60
  h_max: 80
  s_min: 35
  s_max: 255
  v_min: 100
  v_max: 255

#orange
orange:
  h_min: 15
  h_max: 35
  s_min: 80
  s_max: 255
  v_min: 100
  v_max: 255

sizes:
  #enter object area in sq.m.
  red: 0.4875
  blue: 0
  yellow: 0.609
  orange: 0.5
  #enter size tolerance in sq.m.
  tolerance: 1
  #height in metres
  minHeight: 0.000
